definitions/npm/ignore_v4.x.x/test_ignore.js

```diff
@@ -32,3 +32,7 @@ ign.createFilter()(['foo']);
 (ign.ignores('.abc/a.js'): boolean);
 // $ExpectError
 ign.ignores(['foo']);
+
+const paths: string[] = ['.abc/*', '!.abc/d/']
+ign.add(paths)
+ign.filter(paths)
```

Comment:
Oops, missed on thing... Please put the test file into the same flow versioned folder as the implementation. That's our structure moving forward.

Reply:
Like that?